units/provisioning_agent_monitor_deploy.go

```diff
@@ -295,19 +295,19 @@ func (j *agentMonitorDeployJob) agentMonitorOptions(settings *evergreen.Settings
 	agentMonitorParams := []string{
 		binary,
 		"agent",
-		fmt.Sprintf("--api_server='%s'", settings.ApiUrl),
-		fmt.Sprintf("--host_id='%s'", j.host.Id),
-		fmt.Sprintf("--host_secret='%s'", j.host.Secret),
-		fmt.Sprintf("--log_prefix='%s'", filepath.Join(j.host.Distro.WorkDir, "agent")),
-		fmt.Sprintf("--working_directory='%s'", j.host.Distro.WorkDir),
-		fmt.Sprintf("--logkeeper_url='%s'", settings.LoggerConfig.LogkeeperURL),
-		"--cleanup",
-		"monitor",
-		fmt.Sprintf("--log_prefix='%s'", filepath.Join(j.host.Distro.WorkDir, "agent.monitor")),
-		fmt.Sprintf("--client_url='%s'", j.host.ClientURL(settings)),
-		fmt.Sprintf("--client_path='%s'", filepath.Join(j.host.Distro.ClientDir, j.host.Distro.BinaryName())),
+		fmt.Sprintf("--api_server=%s", settings.ApiUrl),
+		fmt.Sprintf("--host_id=%s", j.host.Id),
+		fmt.Sprintf("--host_secret=%s", j.host.Secret),
+		fmt.Sprintf("--log_prefix=%s", filepath.Join(j.host.Distro.WorkDir, "agent")),
+		fmt.Sprintf("--working_directory=%s", j.host.Distro.WorkDir),
+		fmt.Sprintf("--logkeeper_url=%s", settings.LoggerConfig.LogkeeperURL),
+		fmt.Sprintf("--cleanup"),
+		fmt.Sprintf("monitor"),
```

Comment:
I'd not sprintf unless you have format specifiers, (there's fmt.Sprint, which I think just calls .String() or does reasonable stringification for native types, but in this case it really shouldn't be needed.)

Reply:
Removed.